cvmfs/catalog_mgr.h

```diff
@@ -31,6 +31,17 @@ namespace catalog {
 
 const unsigned kSqliteMemPerThread = 1*1024*1024;
 
+static time_t tick(void) {
```

Comment:
I guess this should be maximum be part of the debug build. If not totally removed for default builds